### PR TITLE
ruff: add Spanish translation

### PR DESCRIPTION
**Pull request by Tenvid**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by ikks** [comment]
Hi David, welcome to tldr and thanks for your contributions.

**Comment by Tenvid** [comment]
Thank you! This is my first contribution in a big project, so I'm a bit nervous

File: pages.es/common/ruff.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ruff
+
+> Un linter y formateador de código para Python, escrito en Rust.
+> Más información: <https://docs.astral.sh/ruff/tutorial>.
+
+- Ver documentación para el linter de Ruff:
```
Comment (kant) [inline pages.es/common/ruff.md:6]:
```suggestion
- Vea la documentación para el linter de Ruff:
```

File: pages.es/common/ruff.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`tldr ruff check`
+
+- Ver documentación para el formateador de Ruff:
```
Comment (kant) [inline pages.es/common/ruff.md:10]:
```suggestion
- Vea la documentación para el formateador de Ruff:
```

**Review by kant — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Review by karthik-script — APPROVED** [review]

**Pull request merged**